backend/database/migrations/2017_03_03_075703_add_subpage_fields_to_items_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddSubpageFieldsToItemsTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('items', function (Blueprint $table) {
+            $table->string('backdrop')->nullable();
+            $table->string('slug')->nullable();
+            $table->string('youtube_key')->nullable();
+            $table->string('imdb_id')->nullable();
+            $table->text('overview')->nullable();
+            $table->string('tmdb_rating')->nullable();
+            $table->string('imdb_rating')->nullable();
+        });
+    }
+
+    public function down()
+    {
+        Schema::table('items', function (Blueprint $table) {
+            //
```

Comment:
There has been a debate for a long time about whether `down` methods should be created or not. It happens often that we **can't** create a correct down method (not in this case). 

With Laravel 5.5 there will be a new migration command to reset the complete database without down methods. This should be the prefer path in development.